app/src/main/res/raw/auth_config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "client_id": "ENTER_YOUR_CLIENT_ID",
+  "authorization_user_agent": "WEBVIEW",
```

Comment:
I'd recommend we stick to `DEFAULT` which represents the best end user UX at the time.  What's the thinking behind `WEBVIEW` for this sample?

Reply:
First I want to ask is it `DEFAULT` or `BROWSER`? The docs are not clear which one is device default browser and what is the Chrome customtabs.
If `DEFAULT` is indeed using the device default browser then I agree for the general use-case and I will change it in this PR. If it is Chrome customtabs then I don't agree, not all devices might have Chrome pre-installed by default ([hence why I pushed for the release of version 0.2.0](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/395)).

To explain my use-case (why I choose `WEBVIEW`): https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/426#issuecomment-430722915

Reply:
[Here](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/dev/common/src/main/java/com/microsoft/identity/common/internal/ui/AuthorizationAgent.java) are the AuthorizationAgent options, sorry for sparse docs I'll get that updated.  We're also working on a super in-depth doc on this topic to show all the different user experiences for each option and how to customize the sign in experience. 

MSAL.Android 0.2.0+ never will only use Chrome.  We've removed any code path that results in this based on the data you're citing.  `BROWSER` will always use the default (and CustomTabs if possible, do note that CustomTabs is supported by multiple browsers) and will fallback to other browsers if the default isn't available to perform the sign in (user has disabled it, it's in our future blocked list of browsers that don't behave, etc).  

`DEFAULT` is a bit different in principle, but in practice is exactly the same as `BROWSER`. Abstractly, this refers to our current best experience we can deliver in that release.  The logic may change over time if there develops an improvement for users (which would be reflected in semver changes/highlighted in release notes).

Reply:
Not sure what `DEFAULT` is then if `BROWSER` uses the default browser, very confusing.

So `DEFAULT` (your current recommended) will also work on systems without Chrome (emulators, custom android devices, ...)?

Reply:
@danieldobalian I am not trying to give you a hard time ;) just trying to see what is the best config for a sample app.
To me a sample app doesn't need to have the best UX (if so of course great!) but instead a sample app should be usable/testable on as much devices as possible. So a dev can see that the lib indeed works and after that can dive into the docs and figure out the best config for their own project. If a sample app doesn't work they might just ignore the library and look for an alternative.